README.md

```diff
@@ -63,23 +63,22 @@ Follow these steps to set up your own QUIC implementation:
 
 1. Now, copy the script below into `run_endpoint.sh`, and add commands as
    instructed. Logs should be recorded in `/qnslogs` for them to be available
-   after simulation completion (more on this later). Note that we insert a 10
-   second sleep at the client because we currently have no synchronization, but
-   we need the server and simulator to be running before the client.
+   after simulation completion (more on this later).
 
     ```bash
     #!/bin/bash
     
     # Set up the routing needed for the simulation
     /setup.sh
+    # Wait for the simulator to start up.
+    /wait-for-it.sh sim:57832 -s -t 30
```

Comment:
Do we want to merge this with the `setup.sh` script? Maybe make it a `setup_and_wait.sh` script (any better naming suggestions)?

Reply:
Let's keep them separate, esp since this wait doesn't have to happen on all containers.